### DISCUSSION TITLE
Excel 文件部分单元格预览异常

### DISCUSSION THREAD

**Discussion by DejanHuang** [dotnetcore/BootstrapBlazor#3503] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/44044646/c31c7ac0-5cd2-42ce-a4b1-21982ed88ad6)
![Snipaste_2024-05-14_10-09-35](https://github.com/dotnetcore/BootstrapBlazor/assets/44044646/e57c44c3-139c-465d-b557-9554c6a262ba)
[sample.xlsx](https://github.com/dotnetcore/BootstrapBlazor/files/15301790/sample.xlsx)

**Comment by bb-auto** [comment]
@huangyy21 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.
